callstats-java-sdk/src/main/java/io/callstats/sdk/messages/AuthorizeRequest.java

```diff
@@ -0,0 +1,46 @@
+package io.callstats.sdk.messages;
+
+/**
+*
+* @author Karthik Budigere
+*/
+public class AuthorizeRequest {
```

Comment:
For classes like this "AuthorizeRequest", if you don't want our client to use them, you can remove the "public" access modifier. So that it is only visible within the same package.